src/win/fs.c

```diff
@@ -1221,12 +1221,11 @@ void fs__unlink(uv_fs_t* req) {
 
 void fs__mkdir(uv_fs_t* req) {
   /* TODO: use req->mode. */
-  req->result = _wmkdir(req->file.pathw);
+  req->result = CreateDirectoryW(req->file.pathw, NULL);
   if (req->result == -1) {
```

Comment:
```suggestion
  if (CreateDirectoryW(req->file.pathw, NULL)) {
    SET_REQ_RESULT(req, 0);
  } else {
```

Reply:
great thank you, that seems to fix the cmake build